### PR TITLE
Updated typos in README.md

### PR DESCRIPTION
**Pull request by ritikramuka**
README.md has been updated to correct some typos and grammatical errors.

### CODE REVIEW

**Comment by sr-gi** [comment]
@ritikramuka would you mind going over the rest of the docs? I'm sure there's other small typos there and that way we'll merge them in a single PR.

Otherwise I'll add commits on top of this myself.

**Comment by ritikramuka** [comment]
@sr-gi yep will definitely do that.

**Comment by ritikramuka** [comment]
@sr-gi corrected all of the typos I discovered.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,154 +1,77 @@
...
-
 ```bash
-
 cargo fmt
-
 ```
 
-
-
 In addition, we use [clippy](https://github.com/rust-lang/rust-clippy/) to detect catch common mistakes and improve the code:
```
Comment (sr-gi) [inline CONTRIBUTING.md:12]: to catch detect -> to catch

File: README.md [hunk 2]
```diff
@@ -69,15 +69,15 @@ btc_network = regtest
 
 ### Tower id and signing key
 
-`teos` needs a pair of keys that will serve as tower id and signing key. The former can be used by users to identify the tower, whereas the latter is used by the tower to sign responses. These keys are automatically generated on the first run, and can be refreshed by running `teos` with the `--overwritekey` flag. Notice that once a key is overwritten you won't be able to use the previous key again*.
```
Comment (sr-gi) [inline README.md:72]: `teosd` needs a pair...

Comment (sr-gi) [reply README.md:72]: ... and can be refreshed by running `teosd` ...

File: README.md [hunk 3]
```diff
@@ -53,7 +53,7 @@ teosd --datadir=<path_to_dir>
 
 ### Running TEOS in another network
 
-By default, `rust-teos` runs on `mainnet`. In order to run it on another network you need to change the network parameter in the configuration file or pass the network parameter as a command line option. Notice that if teos does not find a `bitcoind` node running in the same network that it is set to run, it will refuse to run.
```
Comment (sr-gi) [inline README.md:56]: By default, `teosd` ...

Comment (sr-gi) [reply README.md:56]: Notice that if `teosd` does not ...

Comment (sr-gi) [reply README.md:56]: ... change the network where `teosd` will run

File: README.md [hunk 4]
```diff
@@ -69,15 +69,15 @@ btc_network = regtest
...
 
-\* Old keys are actually kept in the tower's database as a fail safe in case you overwrite them by mistake. However, there is no automated way of switching back to and old key. Feel free to open an issue if you overwrote your key by mistake and need support to recover it.
+\* Old keys are actually kept in the tower's database as a fail-safe in case you overwrite them by mistake. However, there is no automated way of switching back to an old key. Feel free to open an issue if you overwrote your key by mistake and need support to recover it.
 
 ## Interacting with a TEOS Instance
 
 You can interact with a `teos` instance (either run by yourself or someone else) by using `teos-cli`. This is an admin tool that has privileged access to the watchtower, and it should therefore only be used within a trusted environment (for example, the same machine).
```
Comment (sr-gi) [inline README.md:78]: You can interact with a `teosd` instance...

File: README.md [hunk 5]
```diff
@@ -69,15 +69,15 @@ btc_network = regtest
...
 
-While `teos-cli` works independently of `teos`, it shares the same configuration file by default, of which it only uses a subset of its settings. The folder can be changed using the `--datadir` command line argument, if desired.
+While `teos-cli` works independently of `teos`, it shares the same configuration file by default, of which it only uses a subset of its settings. The folder can be changed using the `--datadir` command-line argument if desired.
```
Comment (sr-gi) [inline README.md:80]: works independently of `teosd`, it shares ...

**Review by sr-gi — CHANGES_REQUESTED** [review]
Look good overall.

All four commit can be squashed into a single one. There are a few small changes to be included (check comments inline) mainly regarding `teos` -> `teosd`.

File: README.md (hunk 4 above)
Comment (ritikramuka) [reply README.md]: Done

File: README.md (hunk 5 above)
Comment (ritikramuka) [reply README.md]: Done

**Comment by ritikramuka** [comment]
@sr-gi did everything thing you suggested

File: README.md [hunk 6]
```diff
@@ -39,11 +39,11 @@ To change the configuration defaults you can:
 
 - Define a configuration file named `teos.toml` following the template (check [teos/src/conf_template.toml](teos/src/conf_template.toml)) and place it in the `data_dir` (that defaults to `~/.teos/`)
 
-and / or 
+and/or 
 
 - Add some global options when running the daemon (run `teosd -h` for more info).
 
-### Passing command line options to `teosd`
```
Comment (sr-gi) [inline README.md:46]: ### Passing command line options to teosd

File: README.md [hunk 7]
```diff
@@ -53,7 +53,7 @@ teosd --datadir=<path_to_dir>
 
 ### Running TEOS in another network
 
-By default, `rust-teos` runs on `mainnet`. In order to run it on another network you need to change the network parameter in the configuration file or pass the network parameter as a command line option. Notice that if teos does not find a `bitcoind` node running in the same network that it is set to run, it will refuse to run.
+By default, `teosd` runs on `mainnet`. In order to run it on another network, you need to change the network parameter in the configuration file or pass the network parameter as a command-line option. Notice that if teosd does not find a `bitcoind` node running in the same network that it is set to run, it will refuse to run hence change the network where teosd will run.
 
 The configuration file option to change the network where `teos` will run is `btc_network`:
```
Comment (sr-gi) [inline README.md:58]: The configuration file option to change the network where `teosd`

**Review by sr-gi — CHANGES_REQUESTED** [review]
A couple of things that I missed yesterday (see inline)

Also:
- bitcoind -> `bitcoind` (there's some instances of this)
- `rust-teos` comes with -> `teosd` comes with

As a rule of thumb:

- `rust-teos` is the project name and general codebase
- `teosd` refers to the `teos` crate, or the tower daemon
- We use `` when refering to a command / executable 

Feel free to give it another read with that in mind in case I've missed something.

**Comment by sr-gi** [comment]
Hey @ritikramuka, did you address those? No rush if not, I was going through pending PRs to see what could be merged.

**Comment by sr-gi** [comment]
@ritikramuka are you still up for this? I can take over it otherwise.

**Comment by sr-gi** [comment]
Closed via #73

**Pull request closed without merging**